### PR TITLE
[IGA] add output eigen values process

### PR DESCRIPTION
**Pull request by tteschemacher**
**📝 Description**
Adds a process to output the eigen values in a separate form.

### CODE REVIEW

**Comment by manuelmessmer** [comment]
Why does the standard OutputProcess not work?

**Comment by tteschemacher** [comment]
> Why does the standard OutputProcess not work?

The iga output process isn't capable of splitting the eigenvector into its corresponding parts.

File: applications/IgaApplication/python_scripts/output_eigen_values_process.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    if not (isinstance(settings, KratosMultiphysics.Parameters)):
+        raise Exception("Expected input shall be a Parameters object, encapsulating a json string")
+    return IgaEigenValuesOutputProcess(model, settings["Parameters"])
+
+class IgaEigenValuesOutputProcess(KratosMultiphysics.Process):
+
+    def __init__(self, model, params):
+        KratosMultiphysics.Process.__init__(self)
+        print(params)
+        self.process = IGA.OutputEigenValuesProcess(model, params)
```
Comment (philbucher) [inline applications/IgaApplication/python_scripts/output_eigen_values_process.py]:
```suggestion
def Factory(settings, model):
    if not (isinstance(settings, KratosMultiphysics.Parameters)):
        raise Exception("Expected input shall be a Parameters object, encapsulating a json string")
    return IGA.OutputEigenValuesProcess(model, params)
```

much simpler and easier extensible :)

File: applications/IgaApplication/custom_processes/output_eigen_values_process.h [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+{
+
+///@name Kratos Classes
+///@{
+
+/* @class OutputEigenValuesProcess
+ * @ingroup IgaApplication
+ * @brief This class outputs the location of the quadrature points within the local space of the containing geometry. */
+class KRATOS_API(IGA_APPLICATION) OutputEigenValuesProcess
+    : public Process
```
Comment (philbucher) [inline applications/IgaApplication/custom_processes/output_eigen_values_process.h:35]:
```suggestion
    : public OutputProcess
```

File: applications/IgaApplication/custom_processes/output_eigen_values_process.h [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+    /// Destructor.
+    ~OutputEigenValuesProcess() = default;
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    /// Called once before the solution loop and is writing the quadrature domain.
+    void ExecuteFinalize() override;
```
Comment (philbucher) [inline applications/IgaApplication/custom_processes/output_eigen_values_process.h:64]:
```suggestion
    void PrintOutput() override;
```

**Review by philbucher — COMMENTED** [review]
I recommend using a bit newer style :)

File: applications/IgaApplication/python_scripts/output_eigen_values_process.py (hunk 1 above)
Comment (tteschemacher) [reply applications/IgaApplication/python_scripts/output_eigen_values_process.py]: sweet

**Review by rickyaristio — APPROVED** [review]
Looks good to me! Thanks @tteschemacher

**Pull request merged**